public/assets/locales/client.de.json

```diff
@@ -1721,5 +1721,6 @@
   "Buildings": "Gebäude",
   "Water": "Hydrologie",
   "Arrière-fond": "Hintergrund",
-  "Hillshade": "Relief"
+  "Hillshade": "Relief",
+  "main_culture": "Culture"
```

Comment:
does not seem like a correct German translation, where is the source for it ?

Reply:
Thanks for reporting this, I have made a wrong copy/past. Now the client.de.json is up to date.
All translations files have been updated with a manual copy of static files from prod eg. https://map.geoportail.lu/static/7ab2ad3b3c504ca7ba46f9cdec201f72/de.json?cache=7ab2ad3b3c504ca7ba46f9cdec201f72